config/config.json

```diff
@@ -0,0 +1,309 @@
+{
+    "device_ip": "192.168.1.10",
+    "static_transformation": [
+        -0.0195,
+        -0.005,
+        0,
+        0,
+        0,
+        0
+    ],
+    "rgb": {
+        "rtsp_stream_location": "TODO",
```

Comment:
"color" here and "depth" for the second camera  (see [here](https://github.com/Kinovarobotics/ros_kortex_vision/blob/master/launch/kinova_vision_rgbd.launch#L52))